README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Build Status](https://travis-ci.com/retest/recheck-junit-jupiter-extension.svg?branch=master)](https://travis-ci.com/retest/recheck-junit-jupiter-extension)
 [![Latest recheck-junit-jupiter-extension on Maven Central](https://maven-badges.herokuapp.com/maven-central/de.retest/recheck-junit-jupiter-extension/badge.svg?style=flat)](https://mvnrepository.com/artifact/de.retest/recheck-junit-jupiter-extension)
+[![JitPack releaes](https://jitpack.io/v/de.retest/recheck-junit-jupiter-extension.svg)](https://jitpack.io/#de.retest/recheck-junit-jupiter-extension)
```

Comment:
```suggestion
[![JitPack release](https://jitpack.io/v/de.retest/recheck-junit-jupiter-extension.svg)](https://jitpack.io/#de.retest/recheck-junit-jupiter-extension)
```

Note that this will be the `alt` attribute for the corresponding `<img>` element. This text is essentially the same as for the Maven Central badge, maybe sync it (e.g. "Latest release on […]")?